script.js

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable no-undef */
+const createDiv = (() => document.createElement('div'));
+const createLabel = (() => document.createElement('label'));
+let month = '01';
+let year = '01';
+
+function fadePopup() {
+    //fading popout
+    const rectangle = document.querySelector('.popup');
+    const arrow = document.querySelector('.arrowleft');
+    rectangle.classList.add('hide');
+    arrow.classList.add('hide');
+}
+
+const createListBox = ((location, n, array, arrayValues) => {
+    //function for creating a dropdown menu
+    for (let i = 0; i < n; i++) {
+        const option = document.createElement('option');
+        option.value = arrayValues[i];
+        option.text = array[i];
+        location.appendChild(option);
+    }
+});
+
+const getWeekDays = ((nameOfDays) => {
+    //creates the name of days
+    const nameOfDaysArray = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 
+    ' Thursday', 'Friday', 'Saturday'];
+    for (let i = 0; i < 7; i++) {
+        const labelDayName = document.createElement('p');
+        labelDayName.classList.add('days');
+        labelDayName.innerHTML = nameOfDaysArray[i];
+        nameOfDays.appendChild(labelDayName);
+    }
+});
+
+function displayCalendar() {
+    clearExistingCalendar();
```

Comment:
do we need to clear the entire existing dates? We can simply hide the ones that are not applicable in the current month while update the others.